50.mender-binary-delta/docs.md

```diff
@@ -7,6 +7,23 @@ shortcode-core:
 github: false
 ---
 
+
+## mender-binary-delta 1.4.1
+
+_Released 09.25.2022_
+
+### Changelogs
+
+#### mender-binary-delta (1.4.1)
+
+New changes in mender-binary-delta since 1.4.0:
+
+##### Features
+
+* Licenses are now available in the package, instead of only
```

Comment:
Hmm, why the bump?

Reply:
In my opinion this is not a feature, but it was marked as such in the commit. I will manually downgrade it to `Other`